### DISCUSSION TITLE
Is there a way to move a field's note next to its title?

### DISCUSSION THREAD

**Discussion by powerseed** [directus/directus#12360] [Archive]
By default, the note is displayed below the field it belongs to, as shown below:

![image](https://user-images.githubusercontent.com/42278237/159944792-87b56896-b523-48ec-9638-ced12b6da43e.png)

I am wondering if there is a way to move it up, next to the field's title?

Can I use Custom Interface to achieve this?

Thanks!

**Comment by benhaynes** [comment]
This is being discussed over here: https://github.com/directus/directus/discussions/9161

**Reply by powerseed** [reply]
@benhaynes 
Thanks for the reply.
So, that means the feature is not in place yet, is that right?

**Reply by benhaynes** [reply]
Correct.
